.github/workflows/ci.yml

```
@@ -12,13 +12,11 @@ jobs:
         os:
           - ubuntu-latest
         ruby:
-          - 2.3.1
-          - 2.4.1
-          - 2.5.9
-          - 2.6.9
           - 2.7.5
           - 3.0.3
-          - 3.1.0
+          - 3.1.4
+          - 3.2.3
+          - 3.3.0
```

[suggestion] Optional. 3.4.2 is now available: 😃 
https://www.ruby-lang.org/en/news/2025/02/14/ruby-3-4-2-released/

```suggestion
          - 3.3.0
          - 3.4.2
```

---

Added v3.4.2 (and replace 3.3.0 with 3.3.7)